routes/transaction.go

```diff
@@ -25,9 +25,21 @@ import (
 	"github.com/pkg/errors"
 )
 
+type TxnStatus string
+
+const (
+	TxnStatusUnconfirmed TxnStatus = "unconfirmed"
+	TxnStatusConfirmed   TxnStatus = "confirmed"
+	// TODO: It would be useful to have one that is "InBlock" or something like that, which
+	// means we'll consider txns that are in unconfirmed blocks but will *not* consider txns
+	// that are in the mempool. It's a kindof middle-ground.
```

Comment:
Yeah this is a good to do and part of the reason I think creating a helper utility for getting the view makes sense such that we can introduce this functionality later. It's actually not too challenging to do an "in block" thing since we have a "get uncommitted tip view" function already on blockchain.